octobot_tentacles_manager/managers/tentacles_init_files_manager.py

```diff
@@ -59,14 +59,30 @@ def _add_tentacle_to_tentacle_type_init_file(init_content, tentacle, init_file):
             init_file_w.write(init_content)
 
 
-def _remove_tentacle_from_tentacle_type_init_file(init_content, tentacle, init_file):
-    if init_content:
+def _remove_tentacle_from_tentacle_type_init_file(tentacle, init_file):
+    init_content_lines = []
+    if isfile(init_file):
+        # load import file
+        with open(init_file, "r") as init_file_r:
+            init_content_lines = init_file_r.readlines()
+    if init_content_lines:
         # remove import line
-        to_remove_line = f"{get_tentacle_import_block(tentacle)}\n"
-        if to_remove_line in init_content:
-            init_content = init_content.replace(to_remove_line, "")
+        to_remove_lines_count = len(get_tentacle_import_block(tentacle).split("\n")) - 2
+        to_remove_start_index = None
+        tentacle_name_identifier = f"'{tentacle.name}'"
+        for index, line in enumerate(init_content_lines):
+            if tentacle_name_identifier in line:
+                if index > 1 and init_content_lines[index - 1] == "\n":
```

Comment:
It would be better to use https://docs.python.org/3/library/os.html#os.linesep instead of `\n`.